packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -478,6 +478,29 @@ describe('Table', () => {
             expect(element.checkValidity()).toBeTrue();
         });
 
+        it('allows row hover styling when not scrolling', async () => {
```

Comment:
I question the value of these tests, particularly because they don't provide full code coverage and they required writing the implementation code in a non-standard way.

Is it possible to send scroll events to the appropriate element within the table rather than having to fake the value of `isScrolling`? We have tests that scroll the table and get the virtualizer to return different data, so it seems like it should be possible to have the virtualizer know that it is scrolling.

Reply:
Yeah, in the PR description, I explained some of my reservations about these tests and the difficulties of writing more complete tests. These did not catch the fact that I missed adding styling for group rows. I can try to mitigate the concern of "writing the implementation in a non-standard way" by using scroll events instead of mocking the virtualizer, though I'm a little worried that approach is more prone to intermittency. But if we have existing tests doing this non-intermittently, then maybe it's fine.